src/Pages/Repositories/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -468,302 +373,261 @@ const AddContent = () => {
+            />
+          </FormGroup>
+          <FormGroup
+            label='Repository type'
+            fieldId='repositoryType'
+            hasNoPaddingTop
+            // labelIcon={
+            //   <Tooltip content='Put important explanation in here: Put important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in herePut important explanation in here'>
+            //     <OutlinedQuestionCircleIcon className='pf-u-ml-xs' color={global_Color_200.value} />
+            //   </Tooltip>
+            // }
+          >
+            <Flex direction={{ default: 'column' }} gap={{ default: 'gap' }}>
+              <Hide hide={isEdit && contentOrigin === ContentOrigin.UPLOAD}>
+                <Radio
+                  isChecked={values.snapshot && values.origin === ContentOrigin.EXTERNAL}
+                  id='snapshot_radio'
+                  label='Snapshotting'
+                  description={
+                    values.snapshot && values.origin === ContentOrigin.EXTERNAL
+                      ? 'Enable snapshotting for an external repository, allowing you to build images with historical snapshots.'
+                      : ''
+                  }
+                  name='snapshot-radio'
+                  onClick={() =>
+                    setValues({ ...values, snapshot: true, origin: ContentOrigin.EXTERNAL })
+                  }
+                />
+                <Radio
+                  isChecked={!values.snapshot && values.origin === ContentOrigin.EXTERNAL}
+                  id='introspect_radio'
+                  label='Introspect only'
+                  description={
+                    !values.snapshot && values.origin === ContentOrigin.EXTERNAL
+                      ? 'Enable only introspection for an external repository, snapshots will not be taken.'
+                      : ''
+                  }
+                  name='introspect-radio'
+                  onClick={() =>
+                    setValues({ ...values, snapshot: false, origin: ContentOrigin.EXTERNAL })
+                  }
+                />{' '}
               </Hide>
-              <Tr isExpanded={createDataLengthOf1 ? undefined : expanded}>
-                <Td
-                  colSpan={4}
-                  className={
-                    createDataLengthOf1 ? classes.singleContentCol : classes.mainContentCol
+              <Hide hide={isInProd || (isEdit && contentOrigin === ContentOrigin.EXTERNAL)}>
+                <Radio
+                  isChecked={isUploadRepo}
+                  id='upload_radio'
+                  label='Upload'
+                  isDisabled={isEdit && contentOrigin === ContentOrigin.UPLOAD}
+                  description={
+                    isUploadRepo
+                      ? 'Create a repository to upload custom content to, snapshots will be taken after every new upload.'
```

Comment:
Mention the use caes:

Create a repository to upload custom content to, snapshots will be taken after every new upload and can be used in content templates.